jdtls.ext/com.microsoft.jdtls.ext.core/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
metadata files are required for PDE projects?

Reply:
Yes, otherwise the generated .project will not have java nature. Don't know why